KsApi/queries/FetchMyBackedProjectsQuery.graphql

```diff
@@ -0,0 +1,17 @@
+query FetchMyBackedProjects($first: Int = null, $after: String = null, $withStoredCards: Boolean = false) {
+  me {
+    backingsCount
```

Comment:
n.B. that this one uses `backingsCount` on `me` whereas the other query uses `totalCount` on `pageInfo`. `totalCount` for whatever reason is deprecated here.